control-plane-operator/hostedclusterconfigoperator/controllers/resources/resources.go

```diff
@@ -2000,34 +2107,56 @@ func shouldCleanupCloudResources(hcp *hyperv1.HostedControlPlane) bool {
 // cleanupResources generically deletes resources of a given type using an optional filter
 // function. The result is a boolean indicating whether resources were found that match
 // the filter and an error if one occurred.
-func cleanupResources(ctx context.Context, c client.Client, list client.ObjectList, filter func(client.Object) bool, force bool) (bool, error) {
+func cleanupResources(ctx context.Context, c client.Client, list client.ObjectList, filter func(client.Object) bool, force bool) (bool, bool, error) {
```

Comment:
you are changing the return signature of this function but their docs remain the same, why?